packages/to-json-schema/src/convertAction.ts

```diff
@@ -191,6 +207,28 @@ export function convertAction(
       break;
     }
 
+    case 'bic':
+    case 'cuid2':
+    case 'decimal':
+    case 'digits': {
+      jsonSchema.pattern = typeToPatternMap[valibotAction.type].source;
```

Comment:
Is `typeToPatternMap` necessary? These actions should include the regex at the `.requirement` key.